src/OrchardCore.Modules/OrchardCore.Resources/Liquid/StyleBlock.cs

```diff
@@ -0,0 +1,173 @@
+            bool? debug = null;
+            string dependsOn = null;
+            string version = null;
+            var at = ResourceLocation.Unspecified;
+
+            Dictionary<string, string> customAttributes = null;
+
+            foreach (var argument in argumentsList)
+            {
+                switch (argument.Name)
+                {
+                    case "name": name = (await argument.Expression.EvaluateAsync(context)).ToStringValue(); break;
+                    case "condition": condition = (await argument.Expression.EvaluateAsync(context)).ToStringValue(); break;
+                    case "culture": culture = (await argument.Expression.EvaluateAsync(context)).ToStringValue(); break;
+                    case "debug": debug = (await argument.Expression.EvaluateAsync(context)).ToBooleanValue(); break;
+                    case "depends_on": dependsOn = (await argument.Expression.EvaluateAsync(context)).ToStringValue(); break;
+                    case "at": Enum.TryParse((await argument.Expression.EvaluateAsync(context)).ToStringValue(), ignoreCase: true, out at); break;
+                    default: (customAttributes ??= new Dictionary<string, string>())[argument.Name] = (await argument.Expression.EvaluateAsync(context)).ToStringValue(); break;
+                }
+            }
+
+            if (!String.IsNullOrEmpty(name))
+            {
+                // Resource required
+
+                var setting = resourceManager.RegisterResource("stylesheet", name);
+
+                if (customAttributes != null)
+                {
+                    foreach (var attribute in customAttributes)
+                    {
+                        setting.SetAttribute(attribute.Key, attribute.Value);
+                    }
+                }
+
+                if (at != ResourceLocation.Unspecified)
+                {
+                    setting.AtLocation(at);
+                }
+                else
+                {
+                    setting.AtLocation(ResourceLocation.Head);
+                }
+
+                if (!String.IsNullOrEmpty(condition))
+                {
+                    setting.UseCondition(condition);
+                }
+
+                if (debug != null)
+                {
+                    setting.UseDebugMode(debug.Value);
+                }
+
+                if (!String.IsNullOrEmpty(culture))
+                {
+                    setting.UseCulture(culture);
+                }
+
+                if (!String.IsNullOrEmpty(version))
+                {
+                    setting.UseVersion(version);
+                }
+
+                // This allows additions to the pre registered style dependencies.
+                if (!String.IsNullOrEmpty(dependsOn))
+                {
+                    setting.SetDependencies(dependsOn.Split(new[] { ',', ' ' }, StringSplitOptions.RemoveEmptyEntries));
+                }
+
+                // TODO: implement styleblock
```

Comment:
As far as i can see, all LGTM

So just remove this comment, and the related comments in the `ScriptTag` and `StyleTag`

**Update**: After one hour of reviewing